tests/e2e/p/workflow.go

```diff
@@ -25,177 +24,163 @@ import (
 )
 
 // PChainWorkflow is an integration test for normal P-Chain operations
-// - Issues an Add Validator and an Add Delegator using the funding address
-// - Exports AVAX from the P-Chain funding address to the X-Chain created address
-// - Exports AVAX from the X-Chain created address to the P-Chain created address
-// - Checks the expected value of the funding address
+// - Issues an AddPermissionlessValidatorTx
+// - Issues an AddPermissionlessDelegatorTx
+// - Issues an ExportTx on the P-chain and verifies the expected balances
+// - Issues an ImportTx on the X-chain and verifies the expected balances
```

Comment:
This description was previously misleading... The functionality in this test isn't changed.